### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by poojitha892** [hushell/pmf_cvpr22#33]
Hi there,
I'm using dino_base_patch16 arch and I'm using my custom dataset which has 3 classes for image classification. Training is done but I'm not able to use the model to predict on a new image.

**Comment by yuhao20** [comment]
> Hi there, I'm using dino_base_patch16 arch and I'm using my custom dataset which has 3 classes for image classification. Training is done but I'm not able to use the model to predict on a new image.

hello, do u know how to inference on a new image now?

**Comment by hushell** [comment]
Yeah... meta-learning is less obvious than normal image classification. For inference, you'll need support set. Perhaps check this inference code as a reference: https://huggingface.co/spaces/hushell/pmf_with_gis/blob/ef5f3c11fbb89bfad864bd01333b8ec6992f2a40/app.py#L87
